checker/manual/creating-a-checker.tex

```diff
@@ -1109,34 +1107,121 @@ \section{The checker class:  Compiler interface\label{writing-compiler-interface
 checker classes for the Interning Checker and the Nullness Checker:
 
 \begin{Verbatim}
-  @TypeQualifiers({ Interned.class, PolyInterned.class, PolyAll.class })
   @SupportedLintOptions({"dotequals"})
   public final class InterningChecker extends BaseTypeChecker { }
 
-  @TypeQualifiers({ Nullable.class, Raw.class, NonNull.class, PolyNull.class, PolyAll.class })
   @SupportedLintOptions({"flow", "cast", "cast:redundant"})
   public class NullnessChecker extends BaseTypeChecker { }
 \end{Verbatim}
 
+\subsection{Indicating supported annotations\label{indicating-supported-annotations}}
+
+A checker must indicate the annotations that it supports (make up its type
+hierarchy), including whether it supports the polymorphic qualifier
+\refqualclass{framework/qual}{PolyAll}.
+
+By default, a checker supports \<PolyAll>, and all annotations located in a
+subdirectory called \<qual> that's located in the same directory as the checker.
+Note that only annotations defined with the \<@Target({ElementType.TYPE\_USE})>
+meta-annotation (and optionally with the additional value of
+\<ElementType.TYPE\_PARAMETER>, but no other \<ElementType> values)
+are automatically considered as supported annotations.
 
-The checker class must indicate the annotations
-that make up the type hierarchy for this checker (including
-polymorphic qualifiers), either via a
-\refqualclass{framework/qual}{TypeQualifiers} annotation or by overriding
-the
+To indicate support for annotations that are located outside of the \<qual>
+subdirectory, annotations that have other \<ElementType> values, or to indicate
+whether a checker supports the polymorphic qualifier
+\refqualclass{framework/qual}{PolyAll}, checker writers can override the
 \refmethodterse{framework/type}{AnnotatedTypeFactory}{createSupportedTypeQualifiers}{--}
-method.  Each argument to \<@TypeQualifiers> or value returned by
-\<createSupportedTypeQualifiers> is a class literal for a type qualifier
-whose definition bears the
-\refqualclass{framework/qual}{TypeQualifier} meta-annotation.
+method in one of the ways shown below.
+
+Note that \code{createSupportedTypeQualifiers()} returns an immutable set of the
+annotation classes supported by a checker.
+
+In total, there are 5 ways to indicate annotations that are supported by a
```

Comment:
This list doesn't match this lead in sentence -- this list isn't "ways to indicate annotations".  Its a list of imperative sentences directed at the checker.  I can't figure out how to fix this sentence.  @mernst suggestions?